projects/elife.yaml

```diff
@@ -145,6 +145,9 @@ defaults:
                 # (currently fixed) number of EC2 nodes
                 desired-capacity: 1
             helm: false
+            external-dns: false
+                # Route53 hosted zone to create subdomains on
+                #domain-filter: elifesciences.org
```

Comment:
is `domain-filter` part of the EKS vocabulary?

Reply:
it comes from https://github.com/helm/charts/tree/master/stable/external-dns#configuration but in the singular form here; more `external-dns` vocabulary as you could use it with GCP and its Cloud DNS for example